### PR TITLE
Wtp rework

### PR DESCRIPTION
**Pull request by mult1fractal**
* redeuced code of main script
* generated now outputfiles
  * which tools were used
  * how many tools match a specific contig
* stub run for all tools
* database fail safe (from osf)
* preparations for R-markdown result_overview.pdf
* new heatmap design with pvalues (as requested by reviewer)
* bug fixes and file handling
 
closes: #124 
closes: #144 
closes: #143 
closes: #142 
closes: #133 marvel discontinued (producing too many files = need too much storage space during run)

### CODE REVIEW

File: workflows/process/checkV/checkV.nf [hunk 1]
```diff
@@ -1,24 +1,25 @@
 process checkV {
-        publishDir "${params.output}/${name}/${category}", mode: 'copy'
```
Comment (replikation) [inline workflows/process/checkV/checkV.nf:2]: would favor params output more. easier and more transparent to change (in a dedicated params section). not a friend of hardcoded output dirs

File: workflows/process/metaphinder/filter_metaphinder.nf [hunk 2]
```diff
@@ -0,0 +1,12 @@
+process filter_metaphinder {
+    label 'ubuntu'
+    input:
+        tuple val(name), path(results) 
+    output:
+        tuple val(name), path("metaphinder_*.tsv")
+    script:
+        """
+        #cat *.list | sort -g -k4,4 | tail -n+2 | awk '{if(\$2=="phage" && \$3>${params.mp_filter}){print \$1}}'  > metaphinder_\${PWD##*/}.txt
```
Comment (replikation) [inline workflows/process/metaphinder/filter_metaphinder.nf:9]: remove comment?

File: workflows/process/metaphinder_own_DB/download_references.nf [hunk 3]
```diff
@@ -0,0 +1,28 @@
+process download_references {
+    label 'noDocker'
+    errorStrategy 'retry'
+    maxRetries 1   
+    if (params.cloudProcess) { publishDir "${params.databases}/references/", mode: 'copy', pattern: "phage_references.fa" }
```
Comment (replikation) [inline workflows/process/metaphinder_own_DB/download_references.nf:5]: * please change it to "storeDir" only. cloudProcess is not needed anymore

File: workflows/process/metaphinder_own_DB/phage_references_blastDB.nf [hunk 4]
```diff
@@ -0,0 +1,31 @@
+process phage_references_blastDB {
+    if (params.cloudProcess) { publishDir "${params.databases}/blast_DB_phage", mode: 'copy', pattern: "blast_database.tar.gz" }
```
Comment (replikation) [inline workflows/process/metaphinder_own_DB/phage_references_blastDB.nf:2]: storeDir. please check all the outputs

Comment (mult1fractal) [reply workflows/process/metaphinder_own_DB/phage_references_blastDB.nf:2]: yes later...
prio was get it to work properly again so people could do the forking

**Review by replikation — APPROVED** [review]
nicely done. some small adjustments regarding ``storeDir` and parameterized publishDirs.

ill check everything as whole and merge after your changes

**Pull request merged**